homework/tests/homework.spec.js

```diff
@@ -0,0 +1,172 @@
+// @ts-nocheck
+const { test, expect } = require('@playwright/test');
+
+const data = [
+  'Prototype',
+  '1',
+  '2',
+  '3',
+  '4',
+  '5',
+  '6',
+  '7',
+  '8',
+  '9'
+]
+
+//Concatenate
+
+data.forEach(version => {
+  test.describe(version + ': Concatenate', () => {
+    test('Concatenating 2 and 3 results in 23', async ({ page }) => {
+      await page.goto('https://testsheepnz.github.io/BasicCalculator');
```

Comment:
Since this is a common step throughout all test cases - you could use beforeEach hook to navigate to this URL